stable/mission-control/values.yaml

```diff
@@ -306,11 +306,17 @@ missionControl:
   ## Mission Control requires a unique mc.key
   # This will be used to encript sensitive data to be stored in database
   ## You can generate one with the command:
-  ## 'openssl rand -hex 16'
-  ## Pass it to helm with '--set missionControl.mcKey=${MC_KEY}'
-  ## IMPORTANT: You should NOT use the example mcKey for a production deployment!
-  mcKey: bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
+  ## 'openssl rand -hex 32'
+  ## Pass it to helm with '--set missionControl.masterKey=${MASTER_KEY}'
+  ## IMPORTANT: You should NOT use the example masterKey for a production deployment!
+  masterKey: bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
+  mcKey: "{{masterKey}}"
```

Comment:
mcKey: "{{masterKey}}" not going to work

Reply:
Removed mcKey from values.yaml